cf/field.py

```diff
@@ -3899,28 +3898,31 @@ def _weights_scale(self, w, scale):
 
         :Parameters:
 
-            w: `Data
+            w: `Data`
+                The weights to be scaled.
 
-            scale: number
+            scale: number or `None`
+                The maximum value of the scaled, If `None` then no
```

Comment:
```suggestion
                The maximum value of the scaled weights. If `None` then no
```

or similar was the intention, I am guessing?